snuba/admin/static/capacity_management/edit_config_modal.tsx

```diff
@@ -50,7 +50,7 @@ function EditConfigModal(props: {
 
   return (
     <Modal show={currentlyEditing} onHide={() => setCurrentlyEditing(false)}>
-      <Modal.Header closeButton>
```

Comment:
Trying to "grep" for the close button in my tests returned this button and the actual close button so it made it harder to test. This also only added a small `x` at the top right of the modal which would be unlikely to be used anyway since there already exists a "Close" button (or just hitting the `esc` key) - so I just removed it instead.